### DISCUSSION TITLE
Can I get an invite to the closed TestFlight beta?

### DISCUSSION THREAD

**Discussion by SecrestTheHuman** [LeoKlaus/plappa#2] [Q&A] [answered]
Idk if this is rude to ask, but here I am asking it. I’ve been looking around for hours for the best option to listen to my audiobooks on my iPhone. I really don’t wanna install Plex because I don’t feel like it. This really feels like the best option. Or using AltStore to download ABS, but I’d rather use this if I can.

**Comment by LeoKlaus** [comment, marked as answer]
Sure, here’s the link: https://testflight.apple.com/join/CPlREvRs

Plappa is still a beta so you might encounter bugs. If you find anything or have suggestions, feel free to let me know!

**Reply by SecrestTheHuman** [reply]
Awesome! Thank you. Could I get another one for my brother who lives with me. I’ll be sure to warn him about potential bugs.

**Reply by LeoKlaus** [reply]
He can just use the same link :)
